staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -67,9 +67,10 @@ const (
 type MatchPolicyType string
 
 const (
-	// Exact means requests should only be sent to the webhook if they exactly match a given rule.
+	// Exact means requests should only be sent to the admission webhook or admission policy if they exactly match a given rule.
 	Exact MatchPolicyType = "Exact"
-	// Equivalent means requests should be sent to the webhook if they modify a resource listed in rules via another API group or version.
+	// Equivalent means requests should be sent to the admission webhook or admission policy if they modify a resource listed
+	// in rules via another API group or version.
```

Comment:
How about:
```suggestion
	// in rules via an equivalent API group or version. For example, `autoscaling/v1` and `autoscaling/v2`
	// HorizontalPodAutoscalers are equivalent: the same set of resources appear via both APIs.
```

Intent here is to avoid people thinking that you can write a rule for a v1 ConfigMap and also have it apply to a v1 Secret, without listing both `resources` within `resourceRules`.